bdd/spec/032_json_spec.sh

```diff
@@ -0,0 +1,45 @@
+Include build_tools.sh
+
+Describe "JSON"
+  Describe "basic construction and printing"
+    before() {
+      sourceToAll "
+        from @std/app import start, print, exit
+        from @std/json import JSON, toJSON, toString, JSONBase, JSONNode, IsObject, Null
+
+        on start {
+          1.0.toJSON().print();
+          true.toJSON().print();
+          'Hello, JSON!'.toJSON().print();
+          [1.0, 2.0, 5.0].toJSON().print();
+          toJSON().print();
```

Comment:
I really like that all base types implement this interface

Reply:
*Almost* all base types. I haven't implemented it for the `int` types yet, though it should be trivial.